app/(main)/account/settings/page.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+'use client';
+import GlobalSpinner from '@/components/GlobalSpinner/GlobalSpinner';
+import Heading from '@/components/Heading/Heading';
+import LinkCustom from '@/components/LinkCustom/LinkCustom';
+import ResetPasswordForm from '@/components/RestPasswordForm/ResetPasswordForm';
+import UpdateEmailForm from '@/components/UpdateEmailForm/UpdateEmailForm';
+import { useAuthContext } from '@/context/AuthContextProvider';
+import { ChevronLeft } from 'lucide-react';
+import { JSX, useEffect, useState } from 'react';
+
+/**
+ * Display user preferences
+ * @returns {JSX.Element} The rendered user preferences component.
+ */
+const AccountSettings = (): JSX.Element => {
+  const [loadingData, setLoadingData] = useState<boolean>(true);
+  const { isSignedIn } = useAuthContext();
+
+  useEffect(() => {
+    if (isSignedIn) {
+      setLoadingData(false);
+    }
+  }, [isSignedIn]);
+
+  return (
+    <>
+      {loadingData ? (
+        <GlobalSpinner />
+      ) : (
+        <section className="mx-auto max-w-5xl pt-10">
```

Comment:
If you add `space-y-10` on the parent container, in this case `section` it will add `margin` between all of it's direct children. Food for thought?